test/unit_tests/test_braket_device.py

```diff
@@ -456,42 +455,28 @@ def test_non_jaqcd_device(name_mock):
 def test_simulator_default_shots():
     """Tests that simulator devices are analytic if ``shots`` is not supplied"""
     dev = _aws_device(wires=2, device_type=AwsDeviceType.SIMULATOR, shots=None)
-    assert dev.shots == 1
-    assert dev.analytic
-
-
-def test_simulator_0_shots():
-    """Tests that simulator devices are analytic if ``shots`` is not supplied"""
-    dev = _aws_device(wires=2, device_type=AwsDeviceType.SIMULATOR, shots=0)
-    assert dev.shots == 1
+    assert dev.shots is None
     assert dev.analytic
 
 
 def test_local_default_shots():
     """Tests that simulator devices are analytic if ``shots`` is not supplied"""
     dev = BraketLocalQubitDevice(wires=2)
-    assert dev.shots == 1
+    assert dev.shots is None
     assert dev.analytic
 
 
 def test_local_0_shots():
```

Comment:
Should rename the test and update the docstring

```suggestion
def test_local_None_shots():
```